themes/V3/5ePHB/style.less

```diff
@@ -789,6 +789,16 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.page:has(.partCover), 
+.monster,
+.noToC,
+.toc { --TOC: exclude; }
```

Comment:
Add `h4, h5, ` to the default exclusions, to be compatible with current behaviour.